pages/common/tcc.md

```diff
@@ -0,0 +1,16 @@
+# tcc
+
+> A tiny C compiler that can run C source files as scripts and otherwise has command line options similar to `gcc`.
+> More information: <https://bellard.org/tcc/tcc-doc.html>.
+
+- Compile and link `a.c` and `b.c` to generate an executable:
+
+`tcc -o {{executable_name}} {{path/to/a.c}} {{path/to/b.c}}`
```

Comment:
Converter TlDr -> CLIP (Command Line Interface Pages) can't recognize any of `{{path/to/a.c}} {{path/to/b.c}}` placeholders. To fix this issue add `_file` suffix right before .c extension.

Note: issue was fixed manually [here](https://github.com/command-line-interface-pages/cli-pages/pull/47).

Reply:
Or it can be replace with `file1` and `file2` instead of `a` and `b`.

Reply:
Sure, or with `{{path/to/source_file.c}}`.

Reply:
I've now implemented those suggestions.